puppet/modules/web/manifests/vhost/deb.pp

```diff
@@ -20,6 +20,7 @@
     # script to handle deployment too
     secure_ssh::receiver_setup { $user:
       user           => $user,
+      homedir        => $home,
```

Comment:
this is just to be explicit, as we define $home here, and `secure_ssh` uses the same "logic" to calculate the homedir